### PR TITLE
feat(FM-Configuration): Optimize communication to access configure-view 

### PR DESCRIPTION
**Pull request by JannisDommer**
The current interface offers the commands 'open_config' or 'load_config' to fetch the URL to the webview, in which a FM can be configured. However, the URL will be transmitted via a request back to the client with the URL instead of sending it directly as the response.
The PR removes this indirection in the backend and also adapts the vscode extension to handle it properly.

### CODE REVIEW

File: uvls/src/main.rs [hunk 1]
```diff
@@ -386,14 +366,20 @@ impl LanguageServer for Backend {
...
         let uri: Url = serde_json::from_value(params.arguments[0].clone()).unwrap();
+        #[derive(Serialize)]
+        struct OpenArgs {
+            uri: String,
+        }
         match params.command.as_str() {
             "uvls/load_config" => {
                 let target = format!("{}/load{}", self.web_handler_uri, uri.path());
-                self.open_url(target).await;
+                let response: serde_json::Value = serde_json::to_value(OpenArgs { uri: target }).expect("Stop bitching me Rust!");
```
[the remark this answers is not in the document]
Comment (felixrieg) [reply uvls/src/main.rs:376]: Also needs line break for the formatter

File: uvls/src/webview.rs [hunk 2]
```diff
@@ -823,7 +823,7 @@ pub async fn ui_main(
 //HTTP server for the configuration interface
 pub async fn web_handler(pipeline: AsyncPipeline, port: u16) {
     info!("Starting web handler");
-    let addr: std::net::SocketAddr = ([127, 0, 0, 1], port).into();
+    let addr: std::net::SocketAddr = ([0, 0, 0, 0], port).into();
```
Comment (felixrieg) [inline uvls/src/webview.rs:826]: Why is that needed?

File: uvls/src/main.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (JannisDommer) [reply uvls/src/main.rs]: Thanks for pointing out, should be fixed now

File: uvls/src/webview.rs (hunk 2 above)
Comment (JannisDommer) [reply uvls/src/webview.rs]: The address will be used later to bind the WebSocket to. Using localhost made it impossible to call from a remote location. However, as the Playground will drop Configuration as a feature, this change is no longer needed and therefore reverted.

**Review by MartinMUU — APPROVED** [review]
LGTM
